twake/frontend/src/app/components/Menus/Menu.scss

```diff
@@ -54,6 +54,10 @@
     color: $grey_dark;
   }
 
+  .menu-text-delicate {
+    color: red !important;
+  }
+
```

Comment:
We already have a danger class for menu items: ".danger", please use this one :)

Reply:
Done